### ISSUE TITLE
extension not working on linux 

### ISSUE THREAD

**Issue by mustafazeb381-123** [software-mansion/react-native-ide#426]
**Description*

**Environment**
- Version of currently used React Native IDE (Extensions tab -> click on React Native IDE extension -> version is located next to the extension name)
- Version of React Native/Expo used in the project.
- Any other versions of tools related to the problem.

**Comment by jakub-gonet** [comment]
Can you please explain what's happening? Also, please attach logs from VSCode (from extension output tab).

**Comment by wjanes** [comment]
it seems that this extension is not compiled for linux. 
It is not a Bug

**Comment by jakub-gonet** [comment]
Right, we only support macOS for now but we plan to support both windows and linux in the future. I created an feature request tracking this – #428.
